src/scroll-timeline-base.js

```diff
@@ -590,7 +593,6 @@ export function getScrollParent(node) {
         return node;
     }
   }
-  return document.scrollingElement;
```

Comment:
I don't think removing this default return is safe. I don't believe that we are guaranteed to reach the `document.scrollingElement` following the getContainingBlock chain. E.g. I think a fixed position element is not technically "contained" by the document.scrollingElement.

Reply:
Yeah..  
I realized it's better to just check if the node is connected or not before following the getContainingBlock chain.